feature/gribi/otg_tests/mpls_in_udp/README.md

```diff
@@ -0,0 +1,395 @@
+# TE-18.1 gRIBI MPLS in UDP Encapsulation with OC QoS scheduler
+
+Create AFT entries using gRIBI to match destination IP address in a
+network-instance.  Encapsulate the matching packets in MPLS in UDP. Configure
+a qos scheduler using gNMI to rate limit / police traffic based on matching
+the destination IP address of a packet input to the DUT.
+
```

Comment:
Can encap all packets received for a VRF and send to a statically configured destination node (NHG), policy forwarding?

Reply:
revised